packages/sandbox/package.json

```diff
@@ -22,35 +22,35 @@
 		}
 	},
 	"devDependencies": {
-		"@babel/core": "7.21.3",
+		"@babel/core": "^7.25.2",
```

Comment:
These versions aren't fixed and will likely cause unexpected breakages as versions become available and shift.

Reply:
fixing